expression/aggregation.go

```diff
@@ -31,17 +31,26 @@ type AggregationFunction interface {
 	// Update during executing.
 	Update(row []types.Datum, groupKey []byte, ctx context.Context) error
 
+	// StreamUpdate updates data using streaming algo.
+	StreamUpdate(row []types.Datum, groupKey []byte, ctx context.Context) (bool, error)
```

Comment:
StreamUpdate calls getStreamContext and returns a bool value standing for whether the current group by item is equal to last group by items.